src/main/kotlin/fi/metatavu/oss/api/impl/pages/PagesApiImpl.kt

```diff
@@ -0,0 +1,109 @@
+package fi.metatavu.oss.api.impl.pages
+
+import fi.metatavu.oss.api.impl.AbstractApi
+import fi.metatavu.oss.api.impl.UserRole
+import fi.metatavu.oss.api.impl.surveys.SurveyController
+import fi.metatavu.oss.api.model.Page
+import fi.metatavu.oss.api.spec.PagesApi
+import io.quarkus.hibernate.reactive.panache.common.runtime.ReactiveTransactional
+import io.smallrye.mutiny.Uni
+import io.smallrye.mutiny.coroutines.asUni
+import io.vertx.kotlin.coroutines.dispatcher
+import javax.annotation.security.RolesAllowed
+import javax.enterprise.context.RequestScoped
+import javax.inject.Inject
+import javax.ws.rs.core.Response
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.async
+import java.util.*
+
+@RequestScoped
+@Suppress("unused")
+@OptIn(ExperimentalCoroutinesApi::class)
+class PagesApiImpl : PagesApi, AbstractApi() {
+
+    @Inject
+    lateinit var pagesController: PagesController
+
+    @Inject
+    lateinit var surveyController: SurveyController
+
+    @Inject
+    lateinit var pagesTranslator: PagesTranslator
+
+    @Inject
+    lateinit var vertx: io.vertx.core.Vertx
+
+    @ReactiveTransactional
+    @RolesAllowed(UserRole.MANAGER.name, UserRole.CONSUMER_DISPLAY.name)
```

Comment:
Do we actually have such user role as CONSUMER_DISPLAY? I though that those screens do not use Keycloak login at all...